src/test/suite/fixtures/recorded/lineNumbers/takeRowFour.yml

```diff
@@ -5,7 +5,7 @@ command:
   partialTargets:
     - type: primitive
       selectionType: line
```

Comment:
So this `selectionType` comes from talon side?  Probably ok but came as a bit of a surprise.  I wonder if we could infer that info somewhere in the extension, either during inference or processing

Reply:
We could enter it extension side if we want. But that is true for several other commands as well. This solution has the upside that we don't add additional code if I don't actually need to. Instead we reuse the functionality of the line selector and the mark actually only selects the first character of the correct line.